docker-compose.yml

```
@@ -348,6 +365,11 @@ services:
 
   mongo:
     image: mongo:${MONGO_VERSION:-8.0}
+    healthcheck:
+      test: echo 'db.runCommand("ping").ok' | mongosh localhost:27017/test --quiet
```

Can invoke `mongosh` directly.
```suggestion
      test: ["CMD", "mongosh", "mongodb://localhost:27017/test", "--quiet", "--eval",  'db.runCommand("ping").ok']
```

---

Will try

---

Working, applied